### ISSUE TITLE
Disable doesn't work

### ISSUE THREAD

**Issue by mm-gmbd** [jcubic/jquery.terminal#197]
Using jquery-terminal 0.8.8 and neither the `.disable()` function nor setting the `enabled: false` option seem to actually disable the terminal. Per the documentation, "If Terminal is enabled it intercept keyboard", which I thought meant that if Terminal is disabled then it **will not** intercept keyboard, i.e. keypresses will not be displayed on the terminal. Also, although it is not described, I would think that disabling the terminal would also cause the flashing prompt indicator to stop flashing (even if the terminal is clicked/focused).

I would like to use the terminal to echo output that I receive from a different server, but **do not want the user to have direct interaction with the terminal** (I know this is a weird use case, but the styling of the terminal is very nice and familiar to many, so that's why I'd like to use it -- although it is loaded with many other useful features).

Can you help me understand what `.disable()` is supposed to accomplish, and whether or not my described functionality could be achieved?

**Comment by jcubic** [comment]
Actually `.disable()` is like blur, if you want to disable keyboard then you can add keypress and keydown functions as options.

I can add code that will not enable terminal on click when it was disabled.

**Comment by mmuelle4** [comment]
After I wrote my comment, I experimented with the keydown function option and noticed that it did indeed disable the keyboard, but that was only if I didn't supply a `function(command, term)` to the first argument of the constructor. I could accomplish what I want this way (except that the cursor still is flashing), but it's a little bit of a workaround for something that could be a nice "out of the box" feature".

Essentially, I'd like a function that disables all listeners for user input and stops the cursor from blinking (but, I would still be able to trigger an `echo`, change the prompt, etc programatically from Javascript). If this was something that could be toggled on/off, that would be great.

Thanks for the response!
